frontend/public/const.js

```diff
@@ -36,3 +36,5 @@ export const RH_OPERATOR_SUPPORT_POLICY_LINK = 'https://access.redhat.com/third-
 
 // Package manifests for the OperatorHub use this label.
 export const OPERATOR_HUB_LABEL = 'openshift-marketplace';
+
+export const CLI_DOWNLOAD_LINK = 'https://mirror.openshift.com/pub/openshift-v4/clients/4.1';
```

Comment:
LGTM, can we use a 'latest' for instead of hardcoding to 4.1?  We'll have to change this after next release right?

Reply:
We don't want a 4.1 cluster linking to a 4.2 (or 4.3 or 4.4...) CLI download. We're looking at providing a download in-cluster that would always have the right version, so hopefully this is a stop-gap solution.